src/services/database.ts

```diff
@@ -0,0 +1,56 @@
+import mysql from 'mysql';
+
+export default class MySQL {
+    private static _instance : MySQL;
+
+    connection : mysql.Connection;
+    connected: boolean = false;
+
+    constructor(){
+
+        this.connection = mysql.createConnection({
```

Comment:
It was good when you was using `createPool`, because basically each time you connect to the database using `CreateConnection`, you must manually call `connection.release()` after execute new query, otherwise the connection you created will still open and it may cause memory Leak on the data server. `CreatePool` does next behavior: `Connect -> getConnection ->  execute -> release Connection`, in a single query sentence